agent/recorder.go

```diff
@@ -408,3 +433,7 @@ func (r *SpanRecorder) addSpan(span tracer.RawSpan) {
 func isTestSpan(span tracer.RawSpan) bool {
 	return span.Tags["span.kind"] == "test"
 }
+
+func isTestPayloadSpan(span PayloadSpan) bool {
+	return span["span.kind"] == "test"
```

Comment:
```suggestion
	return span.Tags["span.kind"] == "test"
```

And do we really need both `isTestSpan` and this one?

Reply:
Ok, I'm removing one of the methods and adapting the code.